### PR TITLE
v2.0.1

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- #1205
- #1203
- #1207
- #1198
- #1208

## Testing
**URL to test:** Staging

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 version: '3.7'
 services:
     hasura:
-        image: hasura/graphql-engine:v2.34.0
+        image: hasura/graphql-engine:v2.35.1
```
Comment (johnclary) [inline moped-database/docker-compose.yml:6]: this will not actually bump the Hasura engine version in production. that is controlled by ECS config :)

**Review by frankhereford — APPROVED** [review]
Looks good - I have gone over the diffs, and this looks like it's ready to release. Please let me know if you'd like a more thorough pass at it, of course!

**Review by mddilley — APPROVED** [review]
Based on previous testing of these PRs, 🚢 🚢 🚢 😎

**Comment by johnclary** [comment]
thanks, y'all! i appreciate all the back and forth on this extended patch release! 🙌

**Pull request merged**